charts/feature-integrations/integrations/cert-manager-values.yaml

```diff
@@ -3,46 +3,54 @@
 # @section -- General Settings
 name: ""
 
+# -- The value of the job label for scraped metrics and logs
+# @section -- General Settings
+jobName: integrations/cert-manager
+
 # -- Namespaces to look for cert-manager instances.
 # @section -- Discovery Settings
 namespaces: []
 
 # -- Discover cert-manager instances based on field selectors.
 # @section -- Discovery Settings
-field_selectors: []
+fieldSelectors: []
 
 # -- Discover cert-manager instances based on label selectors.
 # Will automatically set a matcher for `app.kubernetes.io/name: <name>` unless set here.
 # @section -- Discovery Settings
 labelSelectors: {}
 
-# -- Name of the port to scrape metrics from.
-# @section -- Discovery Settings
-portName: http-metrics
-
-
-# -- How frequently to scrape metrics from Windows Exporter.
-# @default -- `60s`
-# @section -- Scrape Settings
-scrapeInterval:
-
-# -- The value of the job label for scraped metrics.
-# @section -- Scrape Settings
-jobName: integrations/cert-manager
-
-# -- Sets the max_cache_size for cadvisor prometheus.relabel component.
-# This should be at least 2x-5x your largest scrape target or samples appended rate.
-# ([docs](https://grafana.com/docs/alloy/latest/reference/components/prometheus.relabel/#arguments))
-# Overrides global.maxCacheSize
-# @default -- `100000`
-# @section -- Metric Processing Settings
-maxCacheSize:
-
-# Adjustments to the scraped metrics to filter the amount of data sent to storage.
-metricsTuning:
-  # -- Metrics to keep. Can use regular expressions.
-  # @section -- Metric Processing Settings
-  includeMetrics: []
-  # -- Metrics to drop. Can use regular expressions.
+metrics:
+  # -- Whether to enable metrics collection from cert-manager.
+  # @section -- Scrape Settings
+  enabled: true
+
+  # -- Name of the port to scrape metrics from.
+  # @section -- Discovery Settings
+  portName: http-metrics
+
+  # -- How frequently to scrape metrics from Windows Exporter.
+  # @default -- `60s`
+  # @section -- Scrape Settings
```

Comment:
```suggestion
  # @section -- Metrics Settings
```

Reply:
This will also require a `make build` inside of the `feature-integrations` chart.